source/Calamari.Shared/Integration/Packages/Download/NuGetPackageDownloader.cs

```diff
@@ -140,5 +140,15 @@ void CheckWhetherThePackageHasDependencies(PackagePhysicalFileMetadata pkg)
                     string.Join(", ", dependencies.Select(dependency => $"{dependency.Id}")),
                     WhyAmINotAllowedToUseDependencies);
         }
+        
+        static ICredentials GetFeedCredentials(string? feedUsername, string? feedPassword)
```

Comment:
This seems to be repeated a few times, wondering if it is worth sharing?

Reply:
I did consider that, but I don't know if the behaviour **should** actually be the same or if it is only currently the same **because** it was shared (my guess is the latter).
I'm also not sure where the shared version would make sense? the `PackageDownloaderUtils`? But again, that seems to be too general when it doesn't apply to app downloaders and the `*Utils` could end up as a grab bag of dissociated "helpers".